src/customHooks/useLeaderBoard/utils.ts

```diff
@@ -0,0 +1 @@
+export const errorMessage = (errorMessage: String) => "An error occurred: " + errorMessage;
```

Comment:
It would be more apropriate to put "An error occurred: " in strings and to connect them at the error